File: logica-general/ejercicio-014/resoluciones/lucas-pajarito/tatuajes.js
const tatuajes = [
    {
        cliente: "Carlos",
        diseño: "Dragón",
        tintaDisponible: true,
        maquinaFuncionando: true
    },
    {
        cliente: "Laura",
        diseño: "Rosa",
        tintaDisponible: false,
        maquinaFuncionando: true
    },
    {
        cliente: "Miguel",
        diseño: "Lobo",
        tintaDisponible: true,
        maquinaFuncionando: false
    },
    {
        cliente: "Sofía",
        diseño: "Mariposa",
        tintaDisponible: true,
        maquinaFuncionando: true
    },
    {
        cliente: "Andrés",
        diseño: "Calavera",
        tintaDisponible: false,
        maquinaFuncionando: false
    }
];

function diagnosticarTatuajes(tatuajes) {

    tatuajes.forEach(tatuaje => {

        if (!tatuaje.tintaDisponible) {
            console.log(`${tatuaje.cliente}: Error - No hay tinta disponible.`);
        } else if (!tatuaje.maquinaFuncionando) {
            console.log(`${tatuaje.cliente}: Error - La máquina de tatuar presenta una falla.`);
        } else {
            console.log(`${tatuaje.cliente}: Todo listo para realizar el tatuaje.`);
        }

    });

}

diagnosticarTatuajes(tatuajes);

File: logica-matematica/ejercicio-014/resoluciones/lucas-pajarito/tatuajes.py
tatuajes = [
    {
        "cliente": "Carlos",
        "largo": 12,
        "ancho": 8
    },
    {
        "cliente": "Laura",
        "largo": 15,
        "ancho": 10
    },
    {
        "cliente": "Miguel",
        "largo": 20,
        "ancho": 12
    },
    {
        "cliente": "Sofía",
        "largo": 18,
        "ancho": 9
    },
    {
        "cliente": "Andrés",
        "largo": 10,
        "ancho": 6
    }
]


def calcular_area_perimetro(tatuajes):

    print("=== ÁREA Y PERÍMETRO DE TATUAJES ===")

    for tatuaje in tatuajes:

        area = tatuaje["largo"] * tatuaje["ancho"]
        perimetro = 2 * (tatuaje["largo"] + tatuaje["ancho"])

        print(f'Cliente: {tatuaje["cliente"]}')
        print(f'Largo: {tatuaje["largo"]} cm')
        print(f'Ancho: {tatuaje["ancho"]} cm')
        print(f'Área: {area} cm²')
        print(f'Perímetro: {perimetro} cm')
        print("---------------------------------")


calcular_area_perimetro(tatuajes)